Refactor: Improve Swagger Docs description

EfthymiaKostaki opened the pull request:
### Description
Updated Swagger Docs description for including more information about the project. Also deletes default Namespace which is not needed in the project.

Fixes #482

![UI](https://user-images.githubusercontent.com/43841219/78242439-a5242480-74ea-11ea-806a-ff63e93b08b5.png)

### Type of Change:
- User Interface

### How Has This Been Tested?
When the server is running the updates are immediately visible.

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

EfthymiaKostaki committed:
Refactor: Improve Swagger Docs description

isabelcosta commented:
@EfthymiaKostaki happy to see you figured out how to solve this issue :) Can you please include a screenshot of how the Swagger UI description looks like now? As per our contribution guidelines, when you do a change that involves UI change, its required that you include an image on the PR description. Can you please do that?

EfthymiaKostaki commented:
@isabelcosta  Thank you for your comment. I edited my pull request and added a screenshot of the updated description. Is there anything else I could improve?

isabelcosta commented:
I will wait until you do the removal of the Default namespace, and then I can properly review @EfthymiaKostaki

EfthymiaKostaki committed:
fix: default namespace in UI

EfthymiaKostaki commented:
@isabelcosta  Hello, I have updated this pull request with the commit for the default namespace issue.

naveennvrgup reviewed (dismissed):
Checked the PR locally. Fixes the issue as descripted.

isabelcosta commented on app/api/api_extension.py:
```diff
@@ -3,9 +3,13 @@
     version="1.0",
-    description="API documentation for the backend of Mentorship System"
+    description="API documentation for the backend of Mentorship System \n \n"
+    + "The main repository of the Backend System can be found here: https://github.com/anitab-org/mentorship-backend \n \n"
```
Could you add before this line the description of the app?

**Here it is:** Mentorship System is an application that matches women in tech to mentor each other, on career development, through 1:1 relations during a certain period of time.

isabelcosta reviewed (changes requested):
@EfthymiaKostaki this is great, I just suggest adding the little description for the app there for a little more context :)

isabelcosta commented:
@naveennvrgup thank you a lot for reviewing and testing this! I would love to see more of this from contributors :) I really appreciate your testing here. I think I will start adding a label to issues that need testing to guide you all.

EfthymiaKostaki committed:
refactor: add description of the app

EfthymiaKostaki commented:
@naveennvrgup Thank you for your review! I don't know why GitHub shows message I dismissed your technical review, maybe it's because I added a new commit message.

@isabelcosta I have added the description of the app.

isabelcosta commented:
> @naveennvrgup Thank you for your review! I don't know why GitHub shows message I dismissed your technical review, maybe it's because I added a new commit message.

It is because of the new commit, so that is ok ;)

> @isabelcosta I have added the description of the app.

Thank you so much 👏

isabelcosta reviewed (approved):
Great work @EfthymiaKostaki 👏 
Thank you for your contribution!

rpattath commented:
Tested the changes and seeing the updated documentation.

![Screenshot from 2020-04-07 16-10-41](https://user-images.githubusercontent.com/26095715/78714894-970b4500-78ea-11ea-9069-4835e00685bf.png)

isabelcosta commented:
If you wish @rpattath you can leave a review like a comment, without approval. I know that GitHub counts that as a contribution, whereas comments do not count. I would love to these contributions of yours recorded as such.

![image](https://user-images.githubusercontent.com/11148726/78716431-c9cb3280-7916-11ea-9723-6cfbdac97af3.png)

isabelcosta commented:
Very very good @rpattath thank you so much for your testing :D

isabelcosta commented:
@foongminwong could you please review this and if you do merge, can you make sure the commit follows the guideline and includes the id of the PR?

rpattath commented:
> If you wish @rpattath you can leave a review like a comment, without approval. I know that GitHub counts that as a contribution, whereas comments do not count. I would love to these contributions of yours recorded as such.
> 
> ![image](https://user-images.githubusercontent.com/11148726/78716431-c9cb3280-7916-11ea-9723-6cfbdac97af3.png)

I did not know that, thank you for the tip !

isabelcosta commented:
feel free to try that out here if you want and then check on your GitHub profile if you have an activity item listed saying you "Reviewed a pull request"

isabelcosta commented:
@EfthymiaKostaki PR was merged 👏 thank you for your contribution here 🎉
